### ISSUE TITLE
`getTuntapComponentId: not found component id`

### ISSUE THREAD

**Issue by Daksh777** [nextdns/windows#13]
I'm trying to install the NextDNS GUI client on Windows and here's what I get:
![image](https://user-images.githubusercontent.com/43648146/145570399-4bc164d1-5792-4728-9c0c-885333e17b5d.png)

The installer continues to install after the above message and shows this error:
![image](https://user-images.githubusercontent.com/43648146/145570407-a8ae6a53-eb4a-4cbf-9cd0-ea08d8152ded.png)

**Comment by zeeaall** [comment]
Had the same issue last time I tried using NextDNS, after uninstalling the TAP adapter in an attempt to solve a different issue: https://help.nextdns.io/t/m1hp273

**Comment by Daksh777** [comment]
It seems like the TAP adapter used by OpenVPN (VPN was disabled) was causing the issue.
After uninstalling OpenVPN, I could install the app normally and it started working fine.

However, I'm not sure if this should be now considered as a bug with NextDNS client itself.

**Comment by argymeg** [comment]
Same error here, and same situation (OpenVPN installed, but disabled). I would appreciate the option to keep both installed.

**Comment by Fantom974** [comment]
Just got the same errors here. I was using the NextDNS client successfully for a couple of months and after deciding to give a try to a VPN provider (that rely on a TAP device), it stopped working.

After some investigations, I saw that by installing the NextDNS client (v2.2.1 on Windows 10 x64), the setup program was trying to set up a TAP device and then to get its device ID, I guess. But once I have another TAP device that's not meant for NextDNS but for my VPN connection, **_it fails_**.

In the NextDNS dashboard, they also suggest to use YogaDNS (there's a free & a pro plan) as a client. After setting up my NextDNS ID in YogaDNS, it instantly started to work.. Then I turned on my VPN client, which got connected without any error.

Now I'm having my NextDNS config (and my filters !) working with my VPN.

Hope this temporary workaround will be useful to someone.
